app/src/test/resources/keycloak.json

```diff
@@ -0,0 +1,8 @@
+{
+  "realm": "master",
+  "auth-server-url" : "${sso.auth.server.url}",
+  "resource" : "demoapp",
```

Comment:
Still using `demoapp`?

Reply:
That's used to retrieve the token. According to Paulo there are two clients configured: `demoapp` for managing tokens and `secure-booster-endpoint` for accessing the greeting service. Take a look at the sso booster doc capturing the docs changes

Reply:
Aha, OK. I don't pretend I understand it, but whatever.